addons/website_theme_install/models/theme_models.py

```diff
@@ -85,12 +104,15 @@ def _convert_to_base_model(self, website, **kwargs):
         }
         return new_attach
 
+    def _post_process_creation(self, new_rec):
```

Comment:
maybe call it _post_copy similary to https://github.com/odoo/odoo/pull/29748/files#diff-c507a145345335453f2b4e51cdaed7d2R177

Reply:
ps don't rename post_copy to  post_creation since it overridden in all themes.